frappe/public/js/frappe/views/calendar/calendar.js

```diff
@@ -213,8 +213,9 @@ frappe.views.Calendar = Class.extend({
 	},
 	setup_options: function(defaults) {
 		var me = this;
+		var locale = frappe.boot.user.language === 'zh'? "zh-cn": frappe.boot.user.language;
```

Comment:
Why? 
Ideally, this should be handled server-side.

Reply:
To be frank, I have no idea how to handle it server side, can you give me a pointer?

Reply:
You can fix it in `boot.py` where we create the user object. But why is this fix required?

Reply:
@szufisher any update?

Reply:
How about proceed as what I have done in JS?

Reply:
@szufisher How do we proceed with this until we understand why you've done these changes?

This PR doesn't conform to any of the PR standards or explain why any of the line changes were been made. We're essentially guessing why each line was changed and what could be the possible repercussions of these changes. 

While some of these changes make sense to us and we're glad you've contributed them, our understanding of this may be different than your reasoning for adding them. Nevertheless, @surajshetty3416 has only asked why "zh-cn" is being set as the `frappe.boot.user.language` on the client-side. Please help us understand why so that we can get this PR ready to be merged without further delay.

Reply:
so you mean I got to explain or remark in the source code why each change needed?

actually it is pretty straightforward because most of the time I only added the missing transaction function to make the relevant source text translatable. anyway, if my code change is not up the coding standard, I would like to see my changes to be included in other PRs, then I can close this PR.

Reply:
Hey, can you first answer this one query:

Why is "zh-cn" is being set as the frappe.boot.user.language on the client-side when the server sets it to "zh"?

Reply:
Okay, I reverted back